src/service/resources-impl.js

```diff
@@ -685,7 +686,7 @@ export class ResourcesImpl {
     this.prerenderSize_ = this.viewer_.getPrerenderSize();
 
     const firstPassAfterDocumentReady =
-      this.documentReady_ && this.firstPassAfterDocumentReady_;
+      this.documentReady_ && this.firstPassAfterDocumentReady_ && this.ampInitialized_;
```

Comment:
Revert this for now and reapply after cut.

Reply:
whoops, meant to revert it